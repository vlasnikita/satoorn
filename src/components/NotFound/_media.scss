@media only screen and (max-width: 650px) {
  .NotFound{
    padding: 60px;
    width: calc(100% - 120px);
    height: calc(100% - 120px);

    &__header{
      font-size: 1.4rem;

      &_main{
        font-size: 2.4rem;
      }
    }

    &__item{
      font-size: 1.4rem;
    }
  }
}