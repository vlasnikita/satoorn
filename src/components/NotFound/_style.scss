.NotFound{
  padding: 100px;
  width: calc(100% - 200px);
  height: calc(100% - 200px);

  &__header{
    color: black;
    font-family: none;
    font-size: 1.8rem;
    font-weight: normal;

    &_main{
      font-size: 2.8rem;
      font-weight: bold;
    }
  }

  &__list{
    list-style: disc;
    color: black;
  }

  &__item{
    font-family: none;
    font-size: 1.8rem;
  }
}

@import "/media";