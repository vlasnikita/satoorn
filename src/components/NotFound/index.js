import React, { Component } from 'react';

class NotFound extends Component {

    render() {
        return (
            <div className='NotFound'>
                <h2 className="NotFound__header NotFound__header_main">–¢–∞–∫–æ–π —É—Ä–ª –Ω–∞–º –ø–æ–∫–∞ –Ω–µ –∑–∞–≤–µ–∑–ª–∏ ü§∑‚Äç‚ôÇÔ∏è</h2>
                <br/>
                <br/>
                <h3 className="NotFound__header">–ü—Ä–µ–¥–ª–∞–≥–∞–µ–º –ø–µ—Ä–µ–¥–æ—Ö–Ω—É—Ç—å –Ω–µ–º–Ω–æ–≥–æ –∏ –≤–¥–æ—Ö–Ω–æ–≤–∏—Ç—å—Å—è –∏–¥–µ—è–º–∏ –¥–ª—è –±—É–¥—É—â–∏—Ö —á–µ–ª–ª–µ–Ω–¥–∂–µ–π:</h3>
                <ul className="NotFound__list">
                    <li className="NotFound__item"><a target="_blank" href="https://www.youtube.com/watch?v=dQw4w9WgXcQ">–ö–∞–∫ –≤ –ú–∞–ª–∞–π–∑–∏–∏ –∏—Å–ø—ã—Ç—ã–≤–∞—é—Ç –º–∞–ª—å—á–∏–∫–æ–≤ –≤ 16 –ª–µ—Ç</a></li>
                    <li className="NotFound__item"><a target="_blank" href="https://www.youtube.com/watch?v=dQw4w9WgXcQ">–§–∞–Ω–∞—Ç–∫–∞ PewDiePie –ø—Ä–∏–¥—É–º–∞–ª–∞ –≤ —á–µ—Å—Ç—å –Ω–µ–≥–æ —Ç–∞–Ω–µ—Ü</a></li>
                    <li className="NotFound__item"><a target="_blank" href="https://www.youtube.com/watch?v=dQw4w9WgXcQ">–í –†–∏–≥–µ –ø—Ä–æ–≤–µ–ª–∏ –¢–í-–≤–∏–∫—Ç–æ—Ä–∏–Ω—É —Å–æ —Å—Ç—Ä–∏–º–µ—Ä–∞–º–∏</a></li>
                </ul>
            </div>
        );
    }
}

export default NotFound