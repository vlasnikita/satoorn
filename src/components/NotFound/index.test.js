import ShallowRenderer from 'react-test-renderer/shallow';
import React from 'react'
import NotFound from '../NotFound'

it('should render not found page correctly', () => {
    const renderer = new ShallowRenderer();
    const wrapper = renderer.render(<NotFound />)
    expect(wrapper).toMatchSnapshot()
})