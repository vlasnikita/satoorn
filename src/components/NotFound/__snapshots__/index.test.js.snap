// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render not found page correctly 1`] = `
<div
  className="NotFound"
>
  <h2
    className="NotFound__header NotFound__header_main"
  >
    –¢–∞–∫–æ–π —É—Ä–ª –Ω–∞–º –ø–æ–∫–∞ –Ω–µ –∑–∞–≤–µ–∑–ª–∏ ü§∑‚Äç‚ôÇÔ∏è
  </h2>
  <br />
  <br />
  <h3
    className="NotFound__header"
  >
    –ü—Ä–µ–¥–ª–∞–≥–∞–µ–º –ø–µ—Ä–µ–¥–æ—Ö–Ω—É—Ç—å –Ω–µ–º–Ω–æ–≥–æ –∏ –≤–¥–æ—Ö–Ω–æ–≤–∏—Ç—å—Å—è –∏–¥–µ—è–º–∏ –¥–ª—è –±—É–¥—É—â–∏—Ö —á–µ–ª–ª–µ–Ω–¥–∂–µ–π:
  </h3>
  <ul
    className="NotFound__list"
  >
    <li
      className="NotFound__item"
    >
      <a
        href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"
        target="_blank"
      >
        –ö–∞–∫ –≤ –ú–∞–ª–∞–π–∑–∏–∏ –∏—Å–ø—ã—Ç—ã–≤–∞—é—Ç –º–∞–ª—å—á–∏–∫–æ–≤ –≤ 16 –ª–µ—Ç
      </a>
    </li>
    <li
      className="NotFound__item"
    >
      <a
        href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"
        target="_blank"
      >
        –§–∞–Ω–∞—Ç–∫–∞ PewDiePie –ø—Ä–∏–¥—É–º–∞–ª–∞ –≤ —á–µ—Å—Ç—å –Ω–µ–≥–æ —Ç–∞–Ω–µ—Ü
      </a>
    </li>
    <li
      className="NotFound__item"
    >
      <a
        href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"
        target="_blank"
      >
        –í –†–∏–≥–µ –ø—Ä–æ–≤–µ–ª–∏ –¢–í-–≤–∏–∫—Ç–æ—Ä–∏–Ω—É —Å–æ —Å—Ç—Ä–∏–º–µ—Ä–∞–º–∏
      </a>
    </li>
  </ul>
</div>
`;
